apps/contract/contracts/academy-graduation-nft/README.md

```
@@ -0,0 +1,88 @@
+cd ../progress-tracker
+stellar contract build
+cd ../badge-tracker
+stellar contract build
+```
+
+This generates WASM files in `target/wasm32-unknown-unknown/release/` for:
+- `academy_graduation_nft.wasm`
+- `mock_progress_tracker.wasm`
+- `mock_badge_tracker.wasm`
+
+## 🧪 Testing
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)